dbt_platform_helper/utils/validation.py

```diff
@@ -234,10 +234,19 @@ def is_between(value):
 
 PIPELINES_SCHEMA = Schema(
     {
-        Optional("accounts"): list[str],
         Optional("environments"): [
             {
                 "name": str,
+                Optional("accounts"): {
```

Comment:
I'm not convinced that `environments` or `accounts` are optional any more. Need to confirm that.

@lgarvey @antroy-madetech @ejayesh Do any of you know?

Reply:
It might have been for different reasons really, but during my recent work on SSO, not having `accounts` wasn't an option

Reply:
I haven't done anything with pipelines - so can't comment.

Reply:
Well, environments *is* optional, but you'll get a pipeline that just builds the tooling and then nothing.
Accounts is mandatory yes.

Reply:
So environments may as well be mandatory

Reply:
We've made accounts required, but are not sure whether there is still a use case for creating the codebase pipelines without the environment ones existing.